examples/basic/eventcatalog.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   tagline: 'Discover, Explore and Document your Event Driven Architectures',
   organizationName: 'Your Company',
   editUrl: 'https://github.com/boyney123/eventcatalog-demo/edit/master',
+  homepageLink: 'https://eventcatalog.dev/',
```

Comment:
Maybe for the example (this is what is used for local dev) we should remove this, as I think it would be nice to have local dev copy to go to the `/` (default) route back to the homepage of the Catalog.

Reply:
It is optional, so if the `homepageLink` is not set, the default will be `/`

Reply:
Not sure what I can do.

Reply:
Yeah I think we can remove it from the `examples/basic/eventcatalog.config.js` page